queen-application/src/main/java/fr/insee/queen/application/web/exception/ExceptionControllerAdvice.java

```diff
@@ -189,6 +190,12 @@ public ResponseEntity<ApiError> depositProofException(DepositProofException e, W
         return generateResponseError(e, HttpStatus.INTERNAL_SERVER_ERROR, request);
     }
 
+    @ExceptionHandler(MetadataValueNotFoundException.class)
+    public ResponseEntity<ApiError> metadataValueNotFoundException(MetadataValueNotFoundException e, WebRequest request) {
+        log.error(e.getMessage(), e);
```

Comment:
perhaps move log management to the generateResponseError method to limit code duplication

Reply:
I don't want to log all exceptions, as some are trivial